conformance-test/libraryMethods.ts

```diff
@@ -180,15 +184,18 @@ export async function bucketSetStorageClass(bucket: Bucket) {
 
 export async function bucketUploadResumable(bucket: Bucket) {
   if (bucket.instancePreconditionOpts) {
+    delete bucket.instancePreconditionOpts.ifMetagenerationMatch;
```

Comment:
Perhaps a comment here - I'm not sure what happens if `ifMetagenerationMatch` isn't deleted